crates/optimism/node/src/engine.rs

```diff
@@ -105,6 +105,19 @@ impl<Types> EngineValidator<Types> for OpEngineValidator
 where
     Types: EngineTypes<PayloadAttributes = OpPayloadAttributes>,
 {
+    fn validate_execution_requests(
+        &self,
+        requests: &alloy_eips::eip7685::Requests,
+    ) -> Result<(), EngineObjectValidationError> {
+        // according to op spec, execution requests must be empty
+        if !requests.is_empty() {
+            return Err(EngineObjectValidationError::InvalidParams(
+                "non-empty execution requests".to_string().into(),
```

Comment:
Can we make this match the style of the other error messages?
eg, camel case
```rust
"MissingGasLimitInPayloadAttributes".to_string().into(),
```

Reply:
fixed, should we also update these? https://github.com/meyer9/reth/blob/0b4a0dbb10fcb8a405c155cdb6ac058e8813a52d/crates/payload/primitives/src/lib.rs#L386

Reply:
that would make sense